### ISSUE TITLE
YOLObile: Real-Time Object Detection on Mobile Devices via Compression-Compilation Co-Design

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#123]
# TL;DR
In order to achieve both high accuracy and low latency for mobile ,they proposed a block-puhched pruning method that divides each layer into blocks and learns different pruning patterns in each block. They succeeded in increasing the speed while maintaining the accuracy. The GPU is used to compute the convolutional layer, while the CPU is used to compute the other layers, to further improve the performance.

![FPS vs mAP on MS COCO dataset](https://user-images.githubusercontent.com/24524018/93986825-be36cf80-fdc1-11ea-91c7-af54de9b46b5.png)
![convolution](https://user-images.githubusercontent.com/24524018/93986834-c0009300-fdc1-11ea-859f-87719460acf5.png)

# Paper URL
https://arxiv.org/abs/2009.05697

# Submission Dates(yyyy/mm/dd)
2020/09/12

# Authors and institutions
Yuxuan Cai, Hongjia Li, Geng Yuan, Wei Niu, Yanyu Li, Xulong Tang, Bin Ren, Yanzhi Wang
- Northeastern University
- College of William and Mary
- University of Pittsburgh

# Methods

# Results

# Comments
